src/main/java/com/spring/cinema/config/SecurityConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.spring.cinema.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+    @Autowired
+    public void configureGlobal(AuthenticationManagerBuilder managerBuilder) throws Exception {
+        managerBuilder.inMemoryAuthentication()
+                .withUser("user").password(getEncoder().encode("1111")).roles("USER");
```

Comment:
```suggestion
        managerBuilder
                  .inMemoryAuthentication()
                  .withUser("user")
                  .password(getEncoder().encode("1111"))
                  .roles("USER");
```
does this user pass email validation?

Reply:
le'ts use here user that would pass email validation(event though we are not checking it during logging)